src/OrchardCore.Modules/OrchardCore.Media/Views/Admin/MediaApplication.cshtml

```diff
@@ -170,14 +170,19 @@
             <div class="modal-body">
                 <div class="form-group">
                     <label for="input" class="form-control-label" id="input-label">@T["Folder name"]</label>
-                    <input type="text" id="create-folder-name" class="form-control">
+                    <input type="text" id="create-folder-name" class="form-control d-none d-sm-block">
```

Comment:
We should not have two tags with the same id. The "id" tag has to be unique per document.

Reply:
AFAIK one of the elements will be rendered based on the view

Reply:
They are both rendered. And that is causing the issue. Please take a look a the gif:

![two-ids](https://user-images.githubusercontent.com/2589629/64553819-66eeca00-d33a-11e9-9b48-2c697e284fcb.gif)

Reply:
Strange!! I will revise this while this PR is draft

Thanks

Reply:
You should not use the same "id" attribute twice but you can use the "name" attribute twice so that when you are asking for the values from the form it will be sent as an array of values instead.

Reply:
I know, but I expect one of them should be hidden in certain view, perhaps it's a css class issue